content/purgatorio/collect-and-prepare-data/data-collection.md

```diff
@@ -0,0 +1,209 @@
+
+Beware that not in every case you may be able to automate the process, it will all depend on the domain your problem statement is created from and how digitalised it is. But sometimes some data may need to be extracted manually or through human-intervention or through traditional methods. Sometimes it's easier to not digitalise or automate something till the need for it arises, a few manual steps at rate or slightly occassional times is acceptable, if Data Collection isn't performed on a regular-basis. At the end of the day try to strike a balance between need, efficiency and accuracy - during the initialise steps you may be able to discount these a bit but as you do more iterations discipline and precision is important, so efficiency is our friend then.
+
+[Back to the index](#index)
+
+### Resources
+
+These two resources are quite extensive and have been rated by experienced professionals in the industry:
+
+- [👉 Effective Data Collection 👈](https://www.linkedin.com/posts/asif-bhat_data-collection-activity-6625312371869089793-4LrM)
+- [The Ultimate Guide to Effective Data Collection](https://www.linkedin.com/posts/iamsivab_the-ultimate-guide-to-effective-data-collection-activity-6656175779732381697-lv6X)
+
+It would be good to have a glance of them to see if you have not considered something. Such resources can also help validate the things we already know and how we are using them in the context of the problem to hand (and how others have been using them as well).
+
+Also look at this talk on ["Do we know our data, as good as we know our tools?"](https://github.com/neomatrix369/awesome-ai-ml-dl/blob/master/presentations/data/02-devoxx-uk-2019/README.md), it starts of with an introduction to "Data Collection" and then into the next steps in the end-to-end process, also look for additional resources towards the end of the talk/slides and the Appendix section of the slides.
+
+But don't just stick to these build your own accumen and also look for other resources everywhere else.
+
+[Back to the index](#index)
+
+### Conclusions
+
+So now that you have an idea of what to do when you want to start solving a business problem using techniques like Machine Learning and Deep Learning. You can also start thinking about how to do this in a simple manner and keep building on top of it, otherwise you maybe forever busy collecting data but not proceed further from there. And not know the impact or use of the data collected. And thereby not gain the insight needed to answer your query or solve the problem statement.
+
+Maybe another read of the guide, taking notes, creating a mind-map could help give you a better insight into this initial but important step into Data Science.
+
+[Back to the index](#index)
```

Comment:
Just a last tip: maybe, in each paragraph, highlight some keywords with _italic_ or **bold**. This brings readability and it makes the text more "beautiful".

Reply:
I might need more e-ink for *bold* and **italics** ;) :P

Reply:
I think turning some of them into hyperlinks will also be useful. Let me know if I have missed any. I'll start with them right away.

Reply:
Almost done!